### DISCUSSION TITLE
Sono del tutto estraneo al funzionamento

### DISCUSSION THREAD

**Discussion by Officina2Ford1boccea** [community/community#34835] [New to GitHub]
Vorrei capire se posso utilizzare GitHub per migliorare la visibilità della pagina Facebook e Google

**Comment by jfullstackdev** [comment]
I used an online translator for this

![image](https://user-images.githubusercontent.com/47092464/193434495-77cc2a03-cd2d-4f76-bd94-c8306134ac77.png)

for others to see also,

it seems you want to know how GitHub can help you improve more visibility on Facebook or maybe Google page,

actually I answered several questions in the past regarding this, but in my experience, the more consistent you are in your, say, username, the more the web crawler will recognize and index all of those sites related to that username, hence the more visibility
